src/Symfony/Component/Notifier/Bridge/Mercure/README.md

```diff
@@ -0,0 +1,23 @@
+Mercure Notifier
+===================
+
+Provides [Mercure](https://github.com/symfony/mercure) integration for Symfony Notifier.
+
+DSN example
+-----------
+
+```
+MERCURE_DSN=mercure://PUBLISHER_ID?topic=TOPIC
+```
+
+where:
+ - `PUBLISHER_ID` is the Mercure publisher service id
+ - `TOPIC` is the default topic IRI (optional, default: `https://symfony.com/notifier`)
```

Comment:
```suggestion
 - `TOPIC` is the topic IRI (optional, default: `https://symfony.com/notifier`)
```
?

Reply:
Actually, `TOPIC` could be overridden by the `MercureOptions`. That's why I've put "default" here. 
But both make sense to me. Which one do you prefer?

Reply:
It just described the option, that it is optional and has a default is described in the braces, so I would like to get my suggestion comitted